relay-event-normalization/src/event.rs

```diff
@@ -755,9 +755,9 @@ pub fn normalize_performance_score(
                         && !c.optional
                 }) {
                     // All non-optional measurements with a profile weight greater than 0 are
-                    // required to exist on the event. Skip calculating performance scores if
+                    // required to exist on the event. Skip this profile if
                     // a measurement with weight is missing.
-                    break;
+                    continue;
```

Comment:
Do we need to update the `break` on line 776?

Reply:
I think you're right and we also do not want the `break` at the end, we want to go through all profiles for which the event match the condition.